src/components/Facility/DuplicatePatientDialog.tsx

```diff
@@ -39,7 +39,10 @@ const DuplicatePatientDialog = (props: Props) => {
         <div>
           <p className="text-sm leading-relaxed">
             {t("patient_records_found_description")}(
-            <span className="font-bold">{patientList[0].phone_number}</span>)
+            <span className="font-bold">
+              {patientList[0] && patientList[0]?.phone_number}
```

Comment:
if 0th element exists, do we need a null check?
```suggestion
              {patientList[0] && patientList[0].phone_number}
```